webpack.common.js

```diff
@@ -27,6 +28,9 @@ module.exports = {
       },
       {
         test: /\.css$/,
+        include: [
+          path.resolve(__dirname, 'not_exist_path'), // fixes issue when try to import css file
```

Comment:
Just a small change. Better to tell webpack which files has to be parsed. Rather than a `not_exist_path`, including the source folder like ```include: [path.join(__dirname, 'src')],``` would be more apt (which I did in current project). Also you can remove comments. If that doesn't work, try investigating why this issue happened ? (Guess the solution was from https://github.com/webpack-contrib/css-loader/issues/295)

Reply:
changes done